src/openapi/streaming/subscription.js

```diff
@@ -392,15 +446,22 @@ Subscription.prototype.onSubscribe = function(modify) {
  * @param {Object} args - The arguments of modified subscription.
  * @private
  */
-Subscription.prototype.onModify = function(args) {
+Subscription.prototype.onModify = function(args, options) {
 
     if (this.isDisposed) {
         throw new Error('Modifying a disposed subscription - you will not get data');
     }
 
-    this.onUnsubscribe();
-    this.subscriptionData.Arguments = args;
-    this.onSubscribe(true);
+    if (options && options.method === 'PATCH') {

```

Comment:
i'd prefer options.isPatch since there will not be multiple methods and it doesn't require a patch constant